manifest.json

```diff
@@ -37,6 +37,13 @@
       "js": ["supported_extensions.js", "coverage.js", "socorro.js"],
       "css": ["spinner.css", "socorro.css"],
       "run_at": "document_end"
+    },
+    {
+      "matches": ["*://hg.mozilla.org/mozilla-central/file/*",
+                  "*://hg.mozilla.org/mozilla-central/annotate/*"],
+      "js": ["supported_extensions.js", "coverage.js", "dxr-common.js", "hgmo.js"],
```

Comment:
We should move the `injectToggle` function in a separate file, so we don't make hgmo.js overwrite the `applyOverlay` and `removeOverlay` functions but just define them once. But we can do this in a follow-up.